src/openfermion/tests/_example_test.py

```diff
@@ -18,24 +18,52 @@
 import tempfile
 import unittest
 
-import nbformat
 import numpy
+import pytest
 
 from openfermion.config import THIS_DIRECTORY
 
 
+def _module_import(plug):
+    import sys
+    if sys.version_info >= (3, 4):
+        from importlib import util
+        plug_spec = util.find_spec(plug)
+    else:
+        import pkgutil
+        plug_spec = pkgutil.find_loader(plug)
+    if plug_spec is None:
+        return False
+    else:
+        return True
+
+using_nbtools = pytest.mark.skipif(not (_module_import('nbconvert') and _module_import('ipykernel')),
```

Comment:
Minor style comment, but make sure your files pass pep8 checking (line length in the case of the lines below for the extended strings). It's mostly manual checking at the moment, but we will be converting to enforced pylinter checking soon for PR's